src/handbook/operations/vendors.md

```diff
@@ -23,6 +21,8 @@ The invoice should include:
 - Details of services rendered
 - Amount of dollars due
 
+When verifications are finalised, typically within a couple of business days, the vendor will receive an email with details and next steps.
```

Comment:
Just to check, you checked this with this vendor? I didn't know we provided this service :)

Reply:
They forwarded me this email:
<img width="703" alt="Screenshot 2023-10-11 at 20 39 05" src="https://github.com/FlowFuse/website/assets/129537638/2a9d3739-4fd9-4378-b553-b48a6f2ef345">